ShopGo/LoginScreen/LoginVC.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LoginVC.swift
+//  GoToShop
+//
+//  Created by Waqas Shabir on 7/23/23.
+//
+
+import UIKit
+class LoginVC: UIViewController {
+
+    @IBOutlet weak var logBtnLabel: UIButton!
+    @IBOutlet weak var userNameTxt: UITextField!
+    @IBOutlet weak var passwordTxt: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.userNameTxt.leftImage(imageViewNamed: "username")
+        self.passwordTxt.leftImage(imageViewNamed: "password")
```

Comment:
Check if it still works if you can create a didSet for userNameTxt such as:
 ```
@IBOutlet weak var userNameTxt: UITextField! {
didSet {
self.userNameTxt.leftImage(imageViewNamed: "username")
      }
}
```

Reply:
yes it worked.

Reply:
did not need didSet as i have created a func in the extension.